openshift/tektoncd-pipeline-operator
# Release-v0.9.0

Add 0.9.0 operator CSV

Improve Makefile with new targets for release

Add make targets for:
    - test e2e
    - build operator image
    - push image
    - replace image in opeator deployment manifest
    - create new CSV (add to bundle)
    - operator-courier verify operator
    - push operator to quay application registry

Add v0.9.2 payload

Add v0.9.0 CSV to canary channel

Signed-off-by: Nikhil Thomas <nikthoma@redhat.com>
